var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -109,6 +109,11 @@ class Vtk(CMakePackage):
 
     # For finding Fujitsu-MPI wrapper commands
     patch('find_fujitsu_mpi.patch', when='@:8.2.0%fj')
+    # Freetype@2.10.3 no longer exports FT_CALLBACK_DEF, this
+    # patch replaces FT_CALLBACK_DEF with simple extern "C"
+    # See https://gitlab.kitware.com/vtk/vtk/-/issues/18033
+    patch('vtk-freetype-2.10.3-replace-FT_CALLBACK_DEF.patch',
```

Comment:
Can we retrieve the patch from remote and delete it from the repo?

Reply:
Did not realize or forgot patch could do that.
Pushed a commit deleting the local copy of patch and downloading from remote
(Keeping URL in comments, as that is for discussion of issue, not the patch itself)